src/ogt_vox.h

```diff
@@ -829,6 +829,22 @@
         return default_value;
     }
 
+    static bool _vox_dict_get_value_as_bool(const _vox_dictionary* dict, const char* key_to_find, bool default_value) {
+        const char* str = _vox_dict_get_value_as_string(dict, key_to_find, NULL);
+        if (!str)
+            return default_value;
+        return str[0] == '1' ? true : false;
+    }
+
+    static uint32_t _vox_dict_get_value_as_uint32(const _vox_dictionary* dict, const char* key_to_find, uint32_t default_value) {
+        const char* str = _vox_dict_get_value_as_string(dict, key_to_find, NULL);
+        if (!str)
+            return default_value;
+        uint32_t value;
+        _vox_str_scanf(str, "%i", &value);
```

Comment:
the return value of `_vox_str_scanf` should maybe get checked here and if `!= 1` the `default_value` should be returned, no?

There might be a lot of hand crafted vox files flying around and maybe they don't fulfill the format spec by 100%. And if not checking the result value here, the `uint32_t value` should get initialized.

Reply:
good suggestion, will do.